keystorage/aws/src/test/java/tech/pegasys/signers/aws/AwsSecretsManagerTest.java

```diff
@@ -282,27 +305,20 @@ private String createSecret(final Tag tag, String secretValue)
   }
 
   private String createTestSecret(
-      final String tagKey, final String tagVal, final String secretValue) throws Exception {
-    final Tag testSecretTag = Tag.builder().key(tagKey).value(tagVal).build();
+      final String secretName, final String tagKey, final String tagVal, final String secretValue)
+      throws Exception {
```

Comment:
Since this method can now throw an Exception, I think you can remove the try catch in this method and just let createSecret's Exception bubble up